contributors/devel/api-conventions.md

```diff
@@ -563,6 +566,43 @@ Strategic Merge Patch also supports special operations as listed below.
 
 ### List Operations
 
+#### List of Scalars
+
+We have two merge strategies for list of scalars: replace and merge. Default strategy is replace,
+which will preserve the order; while merge strategy works like an unordered set. We call a list
+using merge strategy a merging list.
+
+##### Merging List of Scalars
+
+To add items "a" and "b" in the finalizers list which is a merging list of strings, the patch is:
+
+```yaml
+finalizers:
+  - a
+  - b
+```
+
+To delete items "c" and "d" in a merging list, we use a parallel list.
+The key for the parallel list is "$deleteFromScalarList/\<keyOfScalarList\>".
+The patch is:
+
+```yaml
+$deleteFromScalarList/finalizers:
+  - c
+  - d
+```
+
+To replace "a" with "b" in a merging list, we combine a deletion and an addition. The patch is:
```

Comment:
But this would delete a from where ever it is, and add b at the end, no? Maybe note this explicitly, as I guess it will be surprising otherwise.